documentation/src/main/asciidoc/stories/assembly_managing_heap_space.adoc

```diff
@@ -0,0 +1,10 @@
+[id='managing_heap_space']
+= Managing Heap Space
+:context: managing_heap_space
```

Comment:
This may break your build if you include this file multiple times in one title. You want to change context in the master before including this assembly there.